docs/DeveloperGuide.md

```diff
@@ -149,6 +149,16 @@ The boolean return value of `parseInput()` indicates whether the user has chosen
 
 #### Delete Command
 
+Delete Command removes an exisiting Student in the StudentList.
+
+[DeleteCommandSequence](./UML_Diagrams/deleteCommand.drawio.svg)
```

Comment:
I think you need `!` before the line to render the image. i.e.
`![DeleteCommandSequence](./UML_Diagrams/deleteCommand.drawio.svg)`